xmodule/capa_block.py

```diff
@@ -2206,13 +2298,40 @@ def update_correctness(self):
         new_correct_map = self.lcp.get_grade_from_current_answers(None)
         self.lcp.correct_map.update(new_correct_map)
 
+    def update_correctness_list(self):
+        """
+        Updates the correct map history of the LCP.
+
+        Similar to `update_correctness` method, but operates on each one of the
+        correctness maps in the history of the LCP.
```

Comment:
I see we're removing the line 2147. Is `update_correctness` still needed?

Reply:
This method is no longer used, I will remove it.